configs/openimages/README.md

```diff
@@ -0,0 +1,125 @@
+# Open Images Dataset
+<!-- [DATASET] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+#### Open Images v6
+
+[Open Images](https://storage.googleapis.com/openimages/web/index.html) is a dataset of ~9M images annotated with image-level labels,
```

Comment:
Need to use the new format of README.md

Reply:
OK, I'll fix it in https://github.com/open-mmlab/mmdetection/pull/6331.